vizro-core/docs/pages/user-guides/layouts.md

```diff
@@ -403,10 +410,278 @@ By default, the grid fits all charts/components on the screen. This can lead to
+- `AgGrid`: See the documentation on [Dash - Change Grid Size](https://dash.plotly.com/dash-ag-grid/grid-size)
+- `DataTable`: See the documentation on [Dash - Setting Table Height](https://dash.plotly.com/datatable/height#setting-table-height-with-vertical-scroll)
+- `Card`: See our documentation on [Card - The extra argument](https://vizro.readthedocs.io/en/stable/pages/user-guides/card/#the-extra-argument)
+
+!!! example "Change the width for Graph"
+    === "app.py"
+        ```{.python pycafe-link}
+        import vizro.models as vm
+        from vizro import Vizro
+        import vizro.plotly.express as px
+
+        tips = px.data.tips()
+
+        page = vm.Page(
+            title="Change the width for Graph",
+            layout=vm.Flex(direction="row", gap="40px", wrap=True),
+            components=[vm.Graph(figure=px.violin(tips, y="tip", x="day", color="day", width=400)) for i in range(5)],
+        )
+
+        dashboard = vm.Dashboard(pages=[page])
+        Vizro().build(dashboard).run()
+        ```
+
+    === "app.yaml"
+        ```yaml
+        # Still requires a .py to add data to the data manager and parse YAML configuration
+        # See yaml_version example
+        pages:
+          - components:
+              - figure:
+                  _target_: violin
+                  x: day
+                  y: tip
+                  color: day
+                  data_frame: tips
+                  width: 400
+                type: graph
+              - figure:
+                  _target_: violin
+                  x: day
+                  y: tip
+                  color: day
+                  data_frame: tips
+                  width: 400
+                type: graph
+              - figure:
+                  _target_: violin
+                  x: day
+                  y: tip
+                  color: day
+                  data_frame: tips
+                  width: 400
+                type: graph
+              - figure:
+                  _target_: violin
+                  x: day
+                  y: tip
+                  color: day
+                  data_frame: tips
+                  width: 400
+                type: graph
+              - figure:
+                  _target_: violin
+                  x: day
+                  y: tip
+                  color: day
+                  data_frame: tips
+                  width: 400
+                type: graph
+            layout:
+              type: flex
+              direction: row
+              gap: 40px
+              wrap: true
+            title: Change the width for Graph
+        ```
+
+    === "Result"
+        [![FlexItemSizeArg]][flexitemsizearg]
+
+#### Change size via custom CSS
+
+Custom CSS is often a better choice than setting sizes through component arguments when you need to apply a consistent `height` and/or `width` across multiple elements without repeating code. This is especially helpful for ensuring uniform sizing of all flex items.
```

Comment:
"Better" is a loaded term if you aren't explaining why it's better. Perhaps "preferred" is a good alternative?